### PR TITLE
Ford3 26 - All models with tests

### PR DESCRIPTION
**Pull request by Christiaanvdm**
Changed some projecta tags to ford3

Created all models and tests.

### CODE REVIEW

File: django_project/ford3/models/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from ford3.models.requirements import *
```
Comment (pierrealixt) [inline django_project/ford3/models/__init__.py]: Use singular naming convention.

File: django_project/ford3/models/qualification_entrance_requirement_subjects.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+from django.db import models
+
+
+class QualificationEntranceRequirementSubjects(models.Model):
```
Comment (pierrealixt) [inline django_project/ford3/models/qualification_entrance_requirement_subjects.py]: use singular naming.

File: django_project/ford3/models/qualifications.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    null=False,
+    unique=False,
+    help_text='',
+    max_length=255)
+  long_description = models.TextField(
+    blank=False,
+    null=False,
+    unique=False,
+    help_text='',)
+  NQF_level = models.IntegerField(
```
Comment (pierrealixt) [inline django_project/ford3/models/qualifications.py]: nqf in lowercase

File: django_project/ford3/models/qualifications.py [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    blank=False,
+    null=False,
+    unique=False,
+    help_text='')
+  estimated_annual_fee = models.IntegerField(
+    blank=False,
+    null=False,
+    unique=False,
+    help_text='')
+  campus_id = models.IntegerField(
```
Comment (pierrealixt) [inline django_project/ford3/models/qualifications.py]: this should be a relationship to the Campus model: `models.ForeignKey(Campus, on_delete=models.CASCADE)`

**Comment by timlinux** [comment]
Can you please add ``.DS_Store`` and ``.sync`` to your ``.gitignore`` file so they don't form part of your changeset? Thanks!

**Comment by timlinux** [comment]
Also try to respond to and resolve your peer review comments ASAP so your PR does not languish.

Thanks!

**Review by pierrealixt — APPROVED** [review]

**Comment by timlinux** [comment]
Nice to see this merged @Christiaanvdm !

**Comment by timlinux** [comment]
Lets aim for small atomic changes from here on

**Pull request merged**